opamp/observiq/observiq_client.go

```diff
@@ -429,11 +441,30 @@ func (c *Client) installPackageFromFile(file *protobufs.DownloadableFile, curPac
 			c.logger.Error("OpAMP client failed to set package statuses", zap.Error(err))
 		}
 
-		c.safeSetUpdatingPackage(false)
 		return
 	}
 
-	// TODO: Start Updater
+	if monitorErr := c.updaterManager.StartAndMonitorUpdater(); monitorErr != nil {
+		// Reread package statuses in case Updater changed anything
+		newPackageStatuses, err := c.packagesStateProvider.LastReportedStatuses()
+		if err != nil {
+			c.logger.Error("Failed to read last reported package statuses", zap.Error(err))
```

Comment:
@cpheps another good catch. i don't know what i was thinking at the time. i'll rework this

Reply:
@cpheps actually going back and looking at this i think it was all intentional. this will end up kind of being what bindplane would rather have i think? so no matter what, LastReportedStatuses will return a statuses even if there is an error (barebones with the package name and version). I was setting the status/error message on this, even though there is no "all packages hash" to reference back to. I figured (maybe) it was better to give Bindplane something rather than nothing?

Reply:
That makes sense just want to make sure that `newPackageStatuses` will never be `nil`.